### PR TITLE
Added Brave 0.7.10dev

### PR DESCRIPTION
**Pull request by rafaelrinaldi**
Brave is a web browser focused on speed and privacy.
More info at: https://brave.com

### CODE REVIEW

**Comment by adidalal** [comment]
You can [add an appcast](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md)

File: Casks/brave.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'brave' do
+  version '0.7.10dev'
+  sha256 '92d21d5223369424baff859b5a38bb72d8b5a7c0b9b4ecf7242455047da64d5a'
+
+  url "https://github.com/brave/browser-laptop/releases/download/v#{version}/Brave.dmg"
```
Comment (vitorgalvao) [inline Casks/brave.rb]: Needs a `# github.com is the official download host per the vendor homepage` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 4f6d339ef2f7e2263f9ee45f144a7a63d925051e. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by rafaelrinaldi** [comment]
@vitorgalvao Thanks for being so welcoming! I hope this is useful to someone. First time I've actually contributed to Cask being a user for years.

Cheers :beers:

**Pull request closed without merging**